src/logger.cpp

```diff
@@ -25,8 +25,21 @@ std::ostream& wsrep::log::os_ = std::cout;
 static wsrep::default_mutex log_mutex_;
 wsrep::mutex& wsrep::log::mutex_ = log_mutex_;
 wsrep::log::logger_fn_type wsrep::log::logger_fn_ = 0;
+int wsrep::log::debug_log_level_ = 0;
 
 void wsrep::log::logger_fn(wsrep::log::logger_fn_type logger_fn)
 {
     logger_fn_ = logger_fn;
 }
+
+void wsrep::log::debug_log_level(int debug_log_level)
+{
+    wsrep::unique_lock<wsrep::mutex> lock(mutex_);
+    debug_log_level_ = debug_log_level;
+}
+
+int wsrep::log::debug_log_level()
+{
+    wsrep::unique_lock<wsrep::mutex> lock(mutex_);
```

Comment:
This looks like quite heavy function for reading single integer value. It would be nice go get rid of mutex lock and maybe even make this method inline.

Reply:
Declared variable as std::atomic_int and removing lock.